cmd/ndm_daemonset/probe/mountprobe.go

```diff
@@ -31,11 +33,15 @@ type mountProbe struct {
 	// create, update, delete, deactivate the disk resources or list the probes already registered.
 	Controller      *controller.Controller
 	MountIdentifier *mount.Identifier
+	ep              *epoll.Epoll
+	dest            chan controller.EventMessage
```

Comment:
Can we have full variable names. Dont need to unnecessarily trim the names. Will go for readability than variable name length